lib/puppet/provider/package/chocolatey.rb

```diff
@@ -86,6 +88,10 @@ def self.choco_ver_cmd
     [command(:chocolatey), *args]
   end
 
+  def set_env
+    Puppet::Provider.environment :ChocolateyInstall => PuppetX::Chocolatey::ChocolateyInstall.installpath
```

Comment:
I'm also not quite sure what I was hoping for this versus `ENV['ChocolateyInstall'] = PuppetX::Chocolatey::ChocolateyInstall.install_path`